types/node-random-name/index.d.ts

```diff
@@ -0,0 +1,30 @@
+// Type definitions for node-random-name 1.0
+// Project: https://github.com/cscott/node-random-name
+// Definitions by: Ankan Bhattacharya <https://github.com/Ankan002>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+interface NodeRandomNameArgs {
+    first?: boolean;
+    last?: boolean;
+    gender?: "male" | "female";
+    seed?: string;
+    random?: () => number;
+    male?: boolean;
+    female?: boolean;
+}
+
+declare function random_name(args?: NodeRandomNameArgs): string;
+
+declare namespace random_name {
+    interface NodeRandomNameArgs {
```

Comment:
This shouldn't be duplicated; if you want to have it exported, you should remove the outer one and refer to it as `random_name.NodeRandomNameArgs` or something.

Reply:
We can implement that